OpenKh.WinShell.IMDThumbnail/OpenKh.WinShell.IMDUtilities.csproj

```diff
@@ -0,0 +1,79 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This is the problem on why the build is failing. You are using the old csproj style, while you should use the new sdk csproj style. There are some tools that can convert it, but my strong suggestion to have the cleanest result is to backup all your files for this project BUT the csproj and `AssemblyInfo.cs`. Then create a new .NET Core project, put the files back and add again the missing NuGet packages and project references. The only difference is that you have to change the value of `<TargetFramework>netcoreapp3.1</TargetFramework>` as `net472`.

Reply:
I see, I will do that in that case.